packages/valory/skills/market_manager_abci/behaviours.py

```diff
@@ -39,15 +45,54 @@
 )
 
 
-class UpdateBetsBehaviour(QueryingBehaviour):
+BETS_FILENAME = "bets.json"
+READ_MODE = "r"
+WRITE_MODE = "w"
+
+
+class BetsManagerBehaviour(BaseBehaviour, ABC):
+    """Abstract behaviour responsible for bets management, such as storing, hashing, reading."""
+
+    def __init__(self, **kwargs: Any) -> None:
+        """Initialize `BetsManagerBehaviour`."""
+        super().__init__(**kwargs)
+        self.bets: List[Bet] = []
+        self.bets_filepath: str = os.path.join(self.context.data_dir, BETS_FILENAME)
+
+    def store_bets(self) -> None:
+        """Store the bets to the agent's data dir as JSON."""
+        serialized = serialize_bets(self.bets)
+        if serialized is None:
+            return
+
+        with open(self.bets_filepath, WRITE_MODE) as bets_file:
+            bets_file.write(serialized)
+
+    def read_bets(self) -> None:
```

Comment:
How are you handling errors here?

Reply:
Added error handling in e3ffdc533881069422e42b280ca1217ca60589a2.